### DISCUSSION TITLE
Enhancement: Will Weekend 2 Coming Release?

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#4025] [Q&A] [answered]
### Issue Checklist

- [ ] I have properly named my enhancement
- [ ] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

WEEKEND 2 IS COMING?

**Comment by Hundrec** [comment, marked as answer]
The second Pitstop update will come sometime in December, according to PhantomArcade. (whoops, this actually didn't happen)
This probably won't include Weekend 2.
